### PR TITLE
[Docs] Warnings Cog Guide

### PR DESCRIPTION
**Pull request by Kreusada**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [x] New feature

### Description of the changes

A user guide for the warnings cog. (#1734)

### CODE REVIEW

**Comment by Kreusada** [comment]
Closing and re-opening to re-run actions

File: docs/cog_guides/warnings.rst [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+
+    [p]unwarn <member> <warn_id> [reason]
+
+**Description**
+
+Remove a warning from a member.
+
+**Arguments**
+
+* ``<member>``: The member to warn. |member-input-quotes|
```
Comment (Flame442) [inline docs/cog_guides/warnings.rst]:
```suggestion
* ``<member>``: The member to remove the warn from. |member-input-quotes|
```

File: docs/cog_guides/warnings.rst [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+    [p]warn <member> [points=1] <reason>
+
+**Description**
+
+Warn the user for the specified reason.
+
+**Arguments**
+
+* ``<member>``: The member to warn. |member-input-quotes|
+* ``<points>``: The number of points the warning should be for. If no number is supplied, 1 point will be given. Pre-set warnings disregard this.
```
Comment (Flame442) [inline docs/cog_guides/warnings.rst]:
```suggestion
* ``[points]``: The number of points the warning should be for. If no number is supplied, 1 point will be given. Pre-set warnings disregard this.
```

File: docs/cog_guides/warnings.rst [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+* ``<reason>``: The reason for the warning. This can be a registered reason if it exists or a custom one is created by default.
```
Comment (Flame442) [inline docs/cog_guides/warnings.rst]: This second sentence is a bit awkward, perhaps break it into 2 sentences? You also should specify that custom reasons are only allowed if `[p]warningset allowcustomreasons` is set.

**Review by Flame442 — CHANGES_REQUESTED** [review]
A few issues, but overall good.

**Comment by Kreusada** [comment]
Thanks for the review, I've made some changes

**Review by Flame442 — APPROVED** [review]
All set, thanks!

**Pull request merged**